lib/fill/fill.test.js

```diff
@@ -1,21 +1,31 @@
 import fill from '.'
 
-test('fills array with value', () => {
+test('fills array with a specified value', () => {
```

Comment:
“Value” has abstract and uncountable meaning, so it requires no article before it, so “...with specified value”